src/main/java/igrad/model/requirement/Requirement.java

```diff
@@ -20,6 +20,12 @@ public Requirement(Title title) {
         this.title = title;

```

Comment:
`private final Title title` instead of `private Title title` (should be final), I'll help you change